MainModule/Client/Plugins/Anti_Cheat.lua

```diff
@@ -313,6 +314,14 @@ return function(Vargs)
 							end
 						end
 
+						if startergear then
+							for _, v in ipairs(startergear:GetChildren()) do
```

Comment:
![image](https://github.com/Epix-Incorporated/Adonis/assets/90981615/07addf41-334e-47d8-b1c5-b79fb5a2058c)

Reply:
fuck that was the wrong image